src/commands/staff/mute.ts

```diff
@@ -10,23 +9,40 @@ export default class extends Command {
 	usage = '<user>';
 	runInDM = false;
 
-	permissions(msg: Message): boolean {
-		return staffPerms(msg);
-	}
+	tempPermissions: ApplicationCommandPermissionData[] = [STAFF_PERMS, ADMIN_PERMS];
+
+	options: ApplicationCommandOptionData[] = [
+		{
+			name: 'user',
+			description: 'The user to mute',
+			type: 'USER',
+			required: true
+		}
+	]
+
+	async tempRun(interaction: CommandInteraction): Promise<void> {
+		const user = interaction.options.getUser('user');
+		const member = await interaction.guild.members.fetch(user.id);
+
+		if (!member) {
+			interaction.reply({ content: `Something went wrong. Please contact ${MAINTAINERS} for help.`, ephemeral: true });
```

Comment:
Would recommend also pointing the command runner to manually give the tagged user the Mute role instead.